code/datums/manufacturing.dm

```diff
@@ -1702,6 +1702,48 @@
 	time = 60 SECONDS
 	category = "Component"
 
+/datum/manufacture/syndicate_laws
+	name = "Syndicate Law Module Set"
+	item_requirements = list("metal_dense" = 40)
+	item_outputs = list(/obj/item/aiModule/syndicate/law1, /obj/item/aiModule/syndicate/law2, /obj/item/aiModule/syndicate/law3, /obj/item/aiModule/syndicate/law4)
+	create = 1
+	time = 60 SECONDS
+	category = "Component"
+
+ABSTRACT_TYPE(/datum/manufacture/aiModule)
+/datum/manufacture/aiModule
+	name = "AI Law Module - 'YOU SHOULDNT SEE ME'"
+	item_requirements = list("metal_dense" = 10)
```

Comment:
Ditto re: conductive materials.

Reply:
I do think AI laws should probably take some conductive, though I am just keeping consistency with other laws for the purposes of this PR